keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1824,7 +1754,20 @@ type ResourceClaim struct {
     // the Pod where this field is used. It makes that resource available
     // inside a container.
     Name string
-}
+
+    // A name set in claim.spec.requests[].name.
+    // +optional
+    RequestName string
+
+    // The index of a device in a request which was satisfied by
+    // more than one device. Starts at zero.
+    //
+    // May only be set together with a request name. Setting
+    // a request name and no index makes all devices of
+    // that request available in the container.
+    //
+    // +optional
+    Index *int64
```

Comment:
I don't think we should allow this. I think "request" should be the granularity of assignment for the container. If they want more granularity, create more requests.

Reply:
I don't feel strongly either way.